templates/watcher/config/00-default.conf

```
@@ -0,0 +1,43 @@
+[DEFAULT]
+state_path = /opt/stack/data/watcher
```

In other operators, `/var/lib/<service_name>` path is used, we can use the same. Below are the examples from other operators.
* https://github.com/openstack-k8s-operators/manila-operator/blob/110a2dbd9f71cd46d2d469a97bbeffd97ef8b716/templates/manila/config/00-config.conf#L3
```suggestion
state_path = /var/lib/watcher
```
We have this path in tcib for watcher. https://github.com/openstack-k8s-operators/tcib/blob/d0b9e012bf830cb131d1d0a740d287b52480bf15/container-images/kolla/base/uid_gid_manage.sh#L74

---

nice catch and there are a couple more in this file (/opt/stack is from the devstack setup likely)

---

All /opt/stack paths are specific to devstack and should be avoided.

---

good catch @raukadah I mised those paths, fixed